summary_page.php

```diff
@@ -111,13 +111,13 @@
 $t_total_time		= number_format( $t_total_time / SECONDS_PER_DAY, 2 );
 $t_average_time 	= number_format( $t_average_time / SECONDS_PER_DAY, 2 );
 
-$t_orct_arr = preg_split( '/[\)\/\(]/', lang_get( 'orct' ), -1, PREG_SPLIT_NO_EMPTY );
+$t_summary_header_arr = preg_split( '/[\)\/\(]/', lang_get( 'summary_header' ), -1, PREG_SPLIT_NO_EMPTY );
```

Comment:
Use a simple _explode()_ call, see earlier comment

Reply:
Refactored string split with explode(), feels digging deep...

About "By Project"  table, we want to see the bug ratios of the only one project selected in the whole project view, also there maybe subprojects as well. Perhaps we can change the header title form "By Project" to "Project" which string name I find is "$s_email_project" in the language file, when only one project selected. 
$ grep \'Project\' lang/strings_english.txt 
$s_email_project = 'Project';
Codes in summary_page.php like this:
@@ -153,7 +153,11 @@ print_summary_submenu();
                <table class="table table-hover table-bordered table-condensed table-striped">
                <thead>
                        <tr>
+                       <?php if( 1 == count( $t_project_ids ) ) { ?>
+                               <th class="width-35"><?php echo lang_get( 'email_project' ) ?></th>
+                       <?php } else { ?>
                                <th class="width-35"><?php echo lang_get( 'by_project' ) ?></th>
+                       <?php } ?>
                                <?php echo $t_summary_header ?>
                        </tr>
                </thead>